operators/tokenizer/bpe_kernels.cc

```diff
@@ -682,6 +682,32 @@ void JsonFastTokenizer::UpdateTokenizer(const TokenJsonConfig& config, const jso
     }
   }
 
+  const json added_tokens_decoder = config.added_tokens_decoder;
```

Comment:
Can you just merge the following implementation into config::LoadAddedToken. https://github.com/microsoft/onnxruntime-extensions/blob/30609b74a1175fb3d47ff8224f42d9640e7f563d/operators/tokenizer/tokenizer_jsconfig.hpp#L274?

Reply:
we load `added_tokens_decoder` from the `TokenJsonConfig& config`, so we would need to pass the entire config to LoadAddedTokens as well in that case. i think we should keep it here as this essentially modifies the added_tokens field before passing it to https://github.com/microsoft/onnxruntime-extensions/blob/2893ff3afd915b756befaa8dbb44960f0527fa02/operators/tokenizer/bpe_tokenizer_model.hpp#L287

Note: this is different from https://github.com/microsoft/onnxruntime-extensions/blob/30609b74a1175fb3d47ff8224f42d9640e7f563d/operators/tokenizer/tokenizer_jsconfig.hpp#L274